src/main/java/ai/promoted/java/client/TwoArmExperiment.java

```diff
@@ -0,0 +1,142 @@
+package ai.promoted.java.client;
+
+import ai.promoted.delivery.model.CohortArm;
+import ai.promoted.delivery.model.CohortMembership;
+
+/**
+ * Represents a two arm Experiment configuration.
+ *
+ * WARNING - while ramping up an experiment, do not change
+ * numControlBuckets or numTreatmentBuckets.  This will
+ * likely produce bad results.
+ *
+ * For the buckets, the treatment buckets are after the
+ * control buckets.
+ */
+public class TwoArmExperiment {
+  /**
+   * Name of cohort.
+   */
+  private final String cohortId;
+  
+  private final int cohortIdHash;
+  
+  /**
+   * Number of the numControlBuckets that are active.
+   */
+  private final int numActiveControlBuckets;
+  
+  /**
+   * Number of control buckets.
+   */
+  private final int numControlBuckets;
+  
+  /**
+   * Number of the numTreatmentBuckets that are active.
+   */
+  private final int numActiveTreatmentBuckets;
+  
+  /**
+   * Number of treatment buckets.
+   */
+  private final int numTreatmentBuckets;
+  
+  private final int numTotalBuckets;
+
+  /**
+   * Factory method for a 50/50 experiment.
+   * @return the configured experiment
+   */
+  public static TwoArmExperiment create5050TwoArmExperimentConfig(String cohortId, int controlPercent,
+      int treatmentPercent) {
+    if (controlPercent < 0 || controlPercent > 100) {
```

Comment:
To make it easy to turn off, might be good to allow controlPercent to be zero.  Otherwise, turning off requires more code changes.

Reply:
Ah sorry.  I misread the conditional.  This is fine.